AnswerKingAPI.yml

```diff
@@ -310,120 +310,23 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/AnswerKing.Orders.Order'
+                $ref: '#/components/schemas/Orders.Order.Response'
```

Comment:
its kept this way to be consistant with the other models within Orders i.e. `Orders.Product.Product` and Orders.Category.Product` since they are all in the domain of `Orders`